packages/gestalt/src/Contents.js

```diff
@@ -52,8 +52,10 @@ type Shift = { x: number, y: number };
 
 type EdgeShift = { caret: Shift, flyout: Shift };
 
+/* eslint-disable react/no-unused-prop-types */
```

Comment:
What's up with this? It looks like all of these props are used.

Reply:
Not sure, Flow/eslnt are having a moment, it was previously on line 487 so I just moved it up, it thinks idealDirection is not used because of the way we pass `this.props` I think